src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -38,6 +42,11 @@
   private SSLParameters sslParameters;
   private HostnameVerifier hostnameVerifier;
 
+  private boolean isUDSConnection(String host) {
```

Comment:
Hello @marcosnils , if we use a **relative path**, the current approach this patch implement may have a potential bug that if the `path` is already included in `/etc/hosts`, then we may open a TCP connection instead, maybe we should change the API or just keep it simple using the **abs path**?